### PR TITLE
add IPUPlace_cn.rst and is_compiled_with_ipu_cn.rst

### PR DESCRIPTION
**Pull request by jianghaicheng**
PR types
New features

PR changes
add docs/api/paddle/device/IPUPlace_cn.rst docs/api/paddle/device/is_compiled_with_ipu_cn.rst

Describe
Add new api of "is_compiled_with_ipu" and "IPUPlace()"

https://github.com/PaddlePaddle/Paddle/pull/37840

最新预览链接 http://10.136.157.23:8090/documentation/docs/zh/faq/index_cn.html?reviewVersion=jenkins-doc-review-2-361

### CODE REVIEW

**Comment by qili93** [comment]
文档预览结果

image

image

image

image

File: docs/api/paddle/device/IPUPlace_cn.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+.. py:class:: paddle.device.IPUPlace
+
+
+
+
+
+``IPUPlace`` 是一个设备描述符，指定 ``IPUPlace`` 则模型将会运行在该设备上。
+
+**代码示例**
```
Comment (TCChenlong) [inline docs/api/paddle/device/IPUPlace_cn.rst]: **代码示例**
->
代码示例
:::::::::

File: docs/api/paddle/device/is_compiled_with_ipu_cn.rst [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+-------------------------------
+
+.. py:function:: paddle.device.is_compiled_with_ipu()
+
+
+
+
+检查 ``whl`` 包是否可以被用来在Graphcore IPU上运行模型
+
+返回：bool，支持Graphcore IPU则为True，否则为False。
```
Comment (TCChenlong) [inline docs/api/paddle/device/is_compiled_with_ipu_cn.rst]: 返回：bool，支持Graphcore IPU则为True，否则为False。
->
返回
:::::::::
bool，支持Graphcore IPU则为True，否则为False。

File: docs/api/paddle/device/is_compiled_with_ipu_cn.rst [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+**示例代码**
```
Comment (TCChenlong) [inline docs/api/paddle/device/is_compiled_with_ipu_cn.rst]: 示例代码同上

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**